packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/test/Providers/MrwSerializationTypeDefinitions/SerializationCustomizationTests.cs

```diff
@@ -212,6 +213,36 @@ public async Task CanChangeDictionaryToBinaryData()
             Assert.AreEqual(Helpers.GetExpectedFromFile(), file.Content);
         }
 
+        private static IEnumerable<TestCaseData> ExtensibleEnumCases =>
+        [
+            new TestCaseData(InputPrimitiveType.String),
+            new TestCaseData(InputPrimitiveType.Int32),
+        ];
+
+        [Test]
```

Comment:
did we mean to leave the `[Test]` attribute?

Reply:
Yeah I guess it isn't required if there is also a TestCaseSource attribute?

Reply:
It shouldn't be. It also flags the method as a test method, per [TestCaseSource](https://docs.nunit.org/articles/nunit/writing-tests/attributes/testcasesource.html).